build/project-page.html

```diff
@@ -1,153 +1,187 @@
-              <div class="project-kid grow bgc-grey--30 link-white">
-                <h3 class="mrg-btm--10">Katalog</h3>
-                <p class="caps pdg-10">Omslag, Print</p>
-              </div>
-              <div class="project-kid bgc-grey--25"><img src="../img/project/kajsa-unge_tremedia-tmb.png" alt="layout och illustration till omslag" class="box-img"></div></a><a id="project-wwl" href="project-wwl.html?id=wwl" class="project">
-              <div class="project-kid grow bgc-grey--5 link-white">
-                <h3 class="mrg-btm--10">wwl</h3>
-                <p class="caps pdg-10">Orderhantering</p>
-              </div>
-              <div class="project-kid bgc-grey"><img src="../img/project/kajsa-unge_wwl-tmb.png" alt="webbapp på imac" class="box-img"></div></a>
-            <!-- PROJECT 5 - BLACK--><a id="project-baksida" href="project-baksida.html?id=baksida" class="project">
-              <div class="project-kid grow bgc-grey-20 link-white">
-                <h3 class="mrg-btm--10">Magasin</h3>
-                <p class="caps pdg-10">Omslag-baksida, Print</p>
-              </div>
-              <div class="project-kid bgc-grey-15"><img src="../img/project/kajsa-unge_baksida-tmb.png" alt="omslag annons i cap och design magasin" class="box-img"></div></a>
-          </section>
-        </section>
-        <section id="contact" class="site-container-col bgc-ku-bg text-dark mrg-top-80 mrg-btm-80 pdg-btm-30">
-          <h1 class="h1_max-75">Säg hej</h1>
-          <p class="s-col mrg-top-20"><a href="tel:+46703987208" class="link-pine">+46 (0)70 3987 208</a><a href="mailto:hej@kajsaunge.se?subject=Mail from Kajsa Unges site" class="link-pine">hej@kajsaunge.se</a></p>
-        </section>
-      </main>
-      <footer id="js_stick-to-bottom" class="site-footer">
-        <div class="mrg-left"></div>
-        <div><a href="https://codepen.io/kajsaunge/" target="_blank" class="mrg-left"><img src="../img/kajsa-unge_codepen.svg" title="Codepen" alt="kajsa unge on codepen. Animations" class="img-footer-icons"></a><a href="https://se.linkedin.com/in/kajsaunge" target="_blank" class="mrg-left-90"><img src="../img/kajsa-unge_linked.svg" title="Kajsa Unge on LinkedIn" alt="kajsa unge linkedin" class="img-footer-icons"></a><a href="https://instagram.com/kajsaunge/" target="_blank" class="mrg-left-90"><img src="../img/kajsa-unge_insta.svg" title="Kajsa Unge on Instagram" alt="kajsa unge instagram" class="img-footer-icons"></a><a href="http://randomize.kajsaunge.se/" target="_blank" class="mrg-left-90"><img src="../img/kajsa-unge_randomize.svg" title="Random list generator, a side-project" alt="kajsa unge randomizr" class="img-footer-icons"></a><a href="http://converter.kajsaunge.se/" target="_blank" class="mrg-left-90"><img src="../img/kajsa-unge_unit-converter.svg" title="Convert units, a side-project" alt="kajsa unge converter" class="img-footer-icons"></a>
-        </div>
-        <div class="mrg-right"></div>
-      </footer>
+
```

Comment:
This is the ugly part as all projects is generated and css handles visibility of each project except avtive ones. Yacksi!